src/js/keyboardView.js

```diff
@@ -0,0 +1,78 @@
+import generateDomElement from './generalFunctions';
+import buttonsLayout from './buttonsLayout';
+import getLocalLanguage from './getLocalLanguage';
+
+function createKeyboardView() {
+    const wrapper = generateDomElement(wrapper, 'div', 'wrapper');
```

Comment:
why do you send undefined as the first argument?

Reply:
Done